estuary/models/errata.py

```diff
@@ -53,6 +53,15 @@ def timeline_timestamp(self):
             return self.created_at.strftime('%Y-%m-%dT%H:%M:%SZ')
         return None
 
+    @property
+    def attached_build_times(self):
```

Comment:
Based on how you use this, it seems wasteful to query for all the attached builds just to see if the specific build in the story is attached and has a build time.

How about replacing this with:
```python
rel = advisory.attached_builds.relationship(build)
time_attached = getattr(rel, 'time_attached', None)
```

Reply:
Doing this was more for the front end so that it'd be easier to get a creation time. I'm not sure that I could get it easily elsewhere.

Reply:
Could you please remind me where this is needed in the frontend?

Reply:
For the creation time of the advisory, unless we would just used the created at property for it still, in which case it would no longer matter.